pytorch/training/docker/1.13/py3/Dockerfile.cpu

```diff
@@ -291,14 +288,16 @@ RUN cd /tmp \
   && rm -rf /tmp/*
 
```

Comment:
Did we remove explicit installation of `opencv-python`?

Reply:
Yes the conda package is called `opencv` which has been added to our installation